README.md

```diff
@@ -1,5 +1,9 @@
 # Incorporating TADs into GWAS Analysis - TAD Pathways
 
+Gregory P. Way and Casey S. Greene 2016
```

Comment:
Please add to the "Greene Lab Items" community on zenodo:
https://zenodo.org/communities/greenelab/?page=1&size=20